zkevm-circuits/src/state_circuit.rs

```diff
@@ -44,6 +44,8 @@ pub struct StateConfig {
     sort_keys: SortKeysConfig,
     is_write: Column<Advice>,
     value: Column<Advice>,
+    prev_value: Column<Advice>,
+    committed_value: Column<Advice>,
```

Comment:
Would perhaps add a short description what is `committed_value` and that it serves for reducing the calls into MPT.